Sources/StreamChatUI/ChatChannelList/ChatChannelListCollectionViewCell.swift

```diff
@@ -11,16 +11,16 @@ public typealias ChatChannelListCollectionViewCell = _ChatChannelListCollectionV
 
 /// A `UICollectionViewCell` subclass that shows channel information.
 open class _ChatChannelListCollectionViewCell<ExtraData: ExtraDataTypes>: _CollectionViewCell,
-    UIConfigProvider {
+    ThemeProvider {
     /// The `ChatChannelListItemView` instance used as content view.
-    open private(set) lazy var itemView: _ChatChannelListItemView<ExtraData> = uiConfig
+    open private(set) lazy var itemView: _ChatChannelListItemView<ExtraData> = currentComponents
```

Comment:
What about `sharedComponents`? 🤔

Reply:
I'm supporting a naming discussion! 
I was also thinking about just `components` 
Ideas @VojtaStavik @b-onc ?

Reply:
I like `components` and `appearance` 👍

Reply:
Amazing, thanks guys 🙌🏻